etc/kayobe/kolla/globals.yml

```diff
@@ -26,7 +26,7 @@ kolla_image_tags:
 # Monitoring and alerting related settings
 
 opensearch_heap_size: 8g
-prometheus_cmdline_extras: "--storage.tsdb.retention.time=30d"
+prometheus_cmdline_extras: "--storage.tsdb.retention.time=30d --query.lookback-delta={{ stackhpc_prometheus_query_lookback_delta }}"
```

Comment:
FWIW, I am +1 tweaking this, given we regularly increase the interval for various exporters.